src/service/resources-impl.js

```diff
@@ -253,6 +257,16 @@ export class ResourcesImpl {
     });
 
     this.rebuildDomWhenReady_();
+
+    if (isExperimentOn(this.win, 'layoutbox-invalidate-on-scroll')) {
+      /** @private @const */
+      this.throttledScroll_ = throttle(this.win, e => this.scrolled_(e), 10);
```

Comment:
Is 10 a meaningful throttle threshold? Once per frame at 60fps would be 17.

IMO this should be once per second or so.

Reply:
I extended it to 250ms.